packages/token-bridge-sdk/src/hooks/useArbTokenBridge.ts

```diff
@@ -1377,7 +1411,8 @@ export const useArbTokenBridge = (
       ) {
         return OutgoingMessageState.EXECUTED
       } else {
-        return bridge.getOutGoingMessageState(batchNumber, indexInBatch)
+        return OutgoingMessageState.NOT_FOUND
```

Comment:
So this is just straight up disabling getting the latest state? This seems fine/right, let's just not forget :)